src/backend/distributed/sql/downgrades/citus--11.0-1--10.2-4.sql

```diff
@@ -1,5 +1,7 @@
 -- citus--11.0-1--10.2-4
 
+DROP FUNCTION pg_catalog.citus_internal_add_object_metadata(text, text[], text[], integer, integer);
```

Comment:
Well, not sure if we have a strict rule on this, but I find adding changes to the bottom of these files more convenient when I search the changes backwards.